### DISCUSSION TITLE
airflow dashboard shows this message  Recent requests have been made to /robots.txt. This indicates that this deployment may be accessible to the public internet. This warning can be disabled by setting webserver.warn_deployment_exposure=False in airflow.cfg. Read more about web deployment security here

### DISCUSSION THREAD

**Discussion by wpwiukyjks** [apache/airflow#27684] [Q&A]
I have airflow version 2.3.0and sudedenly i started seeing this message on dashboard

Recent requests have been made to /robots.txt. This indicates that this deployment may be accessible to the public internet. This warning can be disabled by setting webserver.warn_deployment_exposure=False in airflow.cfg. Read more about web deployment security [here](https://airflow.apache.org/docs/apache-airflow/2.3.0/security/webserver.html)

How can i fix this and what does this mean

**Comment by Taragolis** [comment]
That might indicate that you Airflow webserver instance accessible by everyone in the Internet and some web-crawler like Google, Bing and etc tried to read settings crawler settings.

Usual Airflow deployed in private/security network and this would be the best solution.

**Comment by jamesarosen** [comment]
Is it possible to get more information about the requests? It's _possible_ that something on our private network is making these requests, but we don't know what it would be.

**Comment by potiuk** [comment]
So you should check it in whatever way you usually check access to all your web apps.
